packages/core/src/Agentica.ts

```diff
@@ -226,33 +231,54 @@ export class Agentica<Model extends ILlmSchema.Model> {
           body: {
             ...body,
             model: this.props.vendor.model,
+            stream: true,
```

Comment:
Same question. From now on, it will be delivered to stream unconditionally, but if there is a user who wants RESTful API, will it be RESTful for that function only?

Reply:
Therefore, a function is provided to combine the streams.
They use Stream in common, but they write a lot of Utilities about it